src/modules/ipops/doc/ipops_admin.xml

```diff
@@ -883,6 +883,57 @@ if(dns_query("test.com", "xyz"))
 
     </section>
 
+    <section id="ipops.f.ptr_query">
+      <title>
+        <function moreinfo="none">ptr_query(ip, pvid)</function>
+      </title>
+
+      <para>
+		  Store the hostname that correspond to ip
+		  in a config variable $ptrquery(pvid=>hostname).
```

Comment:
You mean a "pseudo variable" not a config variable.

Reply:
The term "config variable" is also already used in the module, but I agree its probably more clear to spell it explictly.

Reply:
You are right. But it doesn't make it more correct... :-) 

xkaraman - you decide if you want to improve the docs or keep it at the current state.

Reply:
Fixed

Reply:
I use both terms config variables and pseudo variables, and I am referring to the same concept. Historically the term pseudo-variables was used from developer's perspective, trying to differentiate them from C-style variables. Over the time, I noticed that config-variables are easier to "understand" what one refers to, specially by those coming new to use Kamailio.

In other words, pseudo-variables tries to define them from technical point of view (i.e., no declaration, no explicit type association, automatically available by loading various modules, some read-only, other read-write). The config-variables indicates where these variables are available/can be used.